homeassistant/components/mqtt/client.py

```diff
@@ -690,6 +691,7 @@ def _process_client_subscriptions() -> list[tuple[int, int]]:
             results = await self.hass.async_add_executor_job(
                 _process_client_subscriptions
             )
+        self._last_subscribe = time.time()
```

Comment:
Technically the last subscribe is when self._mqttc.subscribe() is called.  Though I guess I can understand not setting this multiple times (once for each subscribe).